src/mlagility/analysis/analysis.py

```diff
@@ -169,19 +168,20 @@ def call_benchit(
         _store_traceback(model_info)
 
     else:
-        # Stats that we want to save into the model's state.yaml file
+        # Stats that we want to save into the build's mlagility_stats.yaml file
```

Comment:
I wonder if we should eventually also save the labels as part of this metadata file.

Reply:
Good suggestion!

Reply:
Resolving this discussion to use this logic to gather initial set of results. 
We can address that suggestion as part of a future PR if we find it relevant.